Math/DefaultMathAdapter.php

```diff
@@ -0,0 +1,468 @@
+<?php
+
+declare (strict_types = 1);
+
+namespace Tdn\PhpTypes\Math;
+
+use Tdn\PhpTypes\Exception\DivisionByZeroException;
+use Tdn\PhpTypes\Exception\InvalidNumberException;
+use Tdn\PhpTypes\Type\StringType;
+
+/**
+ * Class DefaultMathLibraryAdapter.
+ */
+class DefaultMathAdapter extends AbstractMathAdapter implements MathAdapterInterface
```

Comment:
Object needs refactoring. Instead of calliing functions directly, pass math library as dependency implementing a MathLibraryInterface.